File: Python/6py 1/drol.py
"""
|--------------------------------------------------------------------------
| Don't rely on luck.
|--------------------------------------------------------------------------
|
"""

from random import randint
class GuessNum:
   def __eq__(self, x):
      return True
guess = GuessNum()


from random import randint
guess = 1


from random import randint,seed
seed(1)
guess = randint(1,100)
seed(1)

File: PHP/6php 1/drol.php
<?php
/*
|--------------------------------------------------------------------------
| Don't rely on luck.
|--------------------------------------------------------------------------
|
*/

class c extends TestCase {}


srand(0);
$guess = rand(1,100);
srand(0);


class myClass extends TestCase {
   public function work() {
      return $guess;
   }
}